### DISCUSSION TITLE
Reactive scaling

### DISCUSSION THREAD

**Discussion by JayGhiya** [spotify/flink-on-k8s-operator#354] [Ideas]
Is application mode part of roadmap? Would also want to integrate keda as part of task managers so that Reactive scaling can happen seamlessly.

**Comment by regadas** [comment]
Hi @JayGhiya.

Application mode has landed on v0.4.0-beta versions. Reactive mode though still needs to be added.

**Reply by JayGhiya** [reply]
@regadas in terms of adding scale subresource can i get some pointers to the crd to enable reactive scaling?

**Reply by davidkennedylr** [reply]
Is there any documentation or guidance that can be provided around the horizontal pod autoscaler or reactive mode?
